frontend/src/app/main/main.component.html

```diff
@@ -4,10 +4,12 @@
         <h5 class="text-secondary-dark">{{'i18n.modules.main.subtitle' | translate}}</h5>
         <img src="/assets/img/main_icon.svg">
         <h6 class="text-secondary-dark">{{'i18n.modules.main.authors' | translate}}</h6>
-        <div id="credit"><span>Icons made by <a href="http://www.flaticon.com/authors/popcorns-arts"
-                                                title="Popcorns Arts">Popcorns Arts</a> from <a
-            href="http://www.flaticon.com" title="Flaticon">www.flaticon.com</a> is licensed by <a
-            href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0"
-            target="_blank">CC 3.0 BY</a></span></div>
+        <div id="credit">
+            <span>Icons made by <a href="http://www.flaticon.com/authors/popcorns-arts"
+                                   title="Popcorns Arts">Popcorns Arts</a> from <a
```

Comment:
The links are white and no longer visible as they now are on the white page itself